smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java

```diff
@@ -61,59 +62,61 @@
     /**
      * Class to control the inputs of the "Business Service Edit"-Window
      */
-    private class BsmAdminPageEditWindow {
+    public static class BsmAdminPageEditWindow {
```

Comment:
Can we make the classes not inner classes and move them to the src package instead?